### ISSUE TITLE
Where is the configscontainer code?

### ISSUE THREAD

**Issue by MostHated** [CoderGamester/com.gamelovers.configscontainer#17]
Hey there, 
    I saw the description of the repo saying "This package helps with persistently maintain the static config data inside a ScriptableObject in the project", which is just what I was looking for, but I only see code for a property drawer for an enum? I assumed the configscontainer was to help with having configuration info within a scriptableobject be available as a static class or similar. I don't really see anything pertaining to scriptableobjects, am I missing something, or did I just misunderstand what this repo was actually for? 

Thanks,
-MH

**Comment by CoderGamester** [comment]
Hello @MostHated.

Thank you for you message.
You are right, this package lost it's purpose some time ago. The initial functionality was to contain static configs into scriptable objects when importing data from Google Sheets or some other data source.
This package kinda of evolved to be a more Data Type format extension with Vector3 serializers or Enum persistant string serializers.
I'm planning to rebrand this package and others from my repo to be more aligned with it's purpose, unfortunately haven't got the time for that so far.

In case you are still interested on a solution to maintain scriptable objects, I moved the configs container to my Google Sheet importer package: https://github.com/CoderGamester/com.gamelovers.configscontainer

My apologies for the lack of documentation on my packages, but in that package you can see the implementation example in the samples folder or you can also check the implementation on my core-game repo https://github.com/CoderGamester/Core-Game (this repo is like a Game Starter for any of my projects or prototypes, I always fork from it).
- Google sheets importers code: https://github.com/CoderGamester/Core-Game/tree/master/Assets/Editor/SheetImporters
- Configs static containers: https://github.com/CoderGamester/Core-Game/tree/master/Assets/Src/Configs
I maintain the config data on general Locator interfaces protected by their interface visibility.

Let me know if this could fit your purposes. If not, let me know how I can help you achiev your solution or explain it to you better 👍

**Comment by MostHated** [comment]
Hey Miguel,  
    No worries, I can certainly understand project evolution (or sometimes in my case, simply getting bored and decided to do something else midway through, lol). I appreciate you taking the time to reply and the links. 

I typically use Odin Inspector in my projects but started a new project and decided to learn to use UIElements and I wanted to have the option of releasing it if I am satisfied with the results and as such cannot use Odin in that case. I, unfortunately, got quite familiar with a lot of the functionality Odin offered, one being something called "GlobalConfig", which you inherit and then it automatically handles all the work of creating a static instance of the SO which is then readily available for use as soon as you open the editor. 

That was how I came across this repo. I don't currently have any plans to use google docs, but the description of configscontainer sounded pretty close to what I was after. I did spend some time last night coming up with a pretty basic solution, but am always open to better solutions, so I will be sure to take a look at what you shared. 👍 

Thanks again,
-MH
